core/src/banking_stage.rs

```diff
@@ -944,6 +942,7 @@ impl BankingStage {
         };
 
         if let Err(recorder_err) = record_transactions_result {
+            inc_new_counter_info!("banking_stage-max_height_reached", 1);
             inc_new_counter_info!(
                 "banking_stage-record_transactions_retryable_record_txs",
```

Comment:
```suggestion
                "banking_stage-max_height_reached_num_to_commit",
```
@ryoqun if you prefer this name I think you can just hit the commit button on this suggestion. Otherwise I'll merge as is first thing tomorrow

Reply:
come to think of it, i like the naming of removed one. thanks for nice suggestion!